content/intro-to-storybook/ember/en/deploy.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Deploy Storybook'
+tocTitle: 'Deploy'
+description: 'Deploy Storybook online with GitHub and Netlify'
+---
+
+In this tutorial we ran Storybook on our development machine. You may also want to share that Storybook with the team, especially the non-technical members. Thankfully, it’s easy to deploy Storybook online.
+
+<div class="aside">
+<strong>Did you setup Chromatic testing earlier?</strong>
+<br/>
+🎉 Your stories are already deployed! Chromatic securely indexes your stories online and tracks them across branches and commits. Skip this chapter and go to the <a href="/ember/en/conclusion">conclusion</a>.
+</div>
+
+## Exporting as a static app
+
+To deploy Storybook we first need to export it as a static web app. This functionality is already built into Storybook and Ember, we just need to make a small change to our `package.json` file in order for both frameworks to be able to work together and successfully deploy our Storybook.
+
+Add the following to your scripts:
+
+```javascript
+
+// package.json
+{
+    "scripts":{
+        "pre-build":"ember build",
+         "deploy-storybook":"run-s pre-build build-storybook",
+    }
+}
+```
+
+Now when you build Storybook via `npm run deploy-storybook`, it will output a static Storybook in the `storybook-static` directory.
+
+## Continuous deploy
+
+We want to share the latest version of components whenever we push code. To do this we need to continuous deploy Storybook. We’ll rely on GitHub and Netlify to deploy our static site. We’re using the Netlify free plan.
+
+### GitHub
+
+If you're following along from the previous testing chapter jump to setting up a repository on GitHub.
+
+When the project was initialized with ember cli, a local repository was already setup for you. At this stage it's safe to add the files to the first commit.
```

Comment:
```suggestion
When the project was initialized with Ember CLI, a local repository was already setup for you. At this stage it's safe to add the files to the first commit.
```

Reply:
It was fixed with the latest commit. Going to mark it as resolved if you don't mind.